primitives/arithmetic/src/biguint.rs

```diff
@@ -18,7 +18,7 @@
 //! Infinite precision unsigned integer for substrate runtime.
 
 use num_traits::Zero;
-use sp_std::{cmp::Ordering, ops, prelude::*, cell::RefCell, convert::TryFrom};
+use sp_std::{cmp::Ordering, ops, prelude::*, cell::RefCell, convert::TryFrom, vec};
```

Comment:
```suggestion
use sp_std::{cmp::Ordering, ops, prelude::*, cell::RefCell, convert::TryFrom};
```

`vec` comes from `prelude`